autoblocks/_impl/testing/run.py

```diff
@@ -173,6 +176,7 @@ async def run_test_case_unsafe(
             testCaseHash=test_case_ctx.hash(),
             testCaseBody=serialize_test_case(test_case_ctx.test_case),
             testCaseOutput=serialize(output),
+            testCaseDurationMs=(time.perf_counter() - start_time) * 1000,
```

Comment:
this isn't exactly accurate b/c it also includes time waiting for the event loop to execute the task but i think should be good enough for our purposes